docs/production/authentication-methods.md

```diff
@@ -321,12 +322,12 @@ or `NO` otherwise. You can configure a mapping for `deactivated` in
 `AUTH_LDAP_USER_ATTR_MAP`. For example, `"deactivated": "nsAccountLock",` is a correct mapping for a
 [FreeIPA](https://www.freeipa.org/) LDAP database.
 
-Disabled users will be immediately unable to log in
-to Zulip, since Zulip queries the LDAP/Active Directory server on
-every login attempt. The user will be fully deactivated the next time
-your `manage.py sync_ldap_user_data` cron job runs (at which point
-they will be forcefully logged out from all active browser sessions,
-appear as deactivated in the Zulip UI, etc.).
+Disabled users will be immediately unable to log in to Zulip, since
+Zulip queries the LDAP/Active Directory server on every login
+attempt. The user will be fully deactivated the next time you run
+`manage.py sync_ldap_user_data` (at which point they will be
+forcefully logged out from all active browser sessions, appear as
+deactivated in the Zulip UI, etc.).
```

Comment:
Probably worth a clarification that this is true for login **via LDAP**? If another backend is still enabled, then the user can login - until the full deactivation happens during the sync job.

The rest of this doc update sounds right to me

Reply:
Good point, did a bit of a broader tweak to that effort. Thanks for the review!